themes/default/layouts/page/get-started.html

```diff
@@ -0,0 +1,57 @@
+{{ define "hero" }}
+    {{ partial "hero.html" (dict "title" .Params.title) }}
+{{ end }}
+
+{{ define "main" }}
+    <section class="container mx-auto mt-20">
+        <div class="shape-background get-started-install-section">
+            <div class="shape-container">
+                <div class="circle-right"></div>
+            </div>
+        </div>
+        <div class="text-center mb-8 bg-violet-200 w-12 h-12 rounded-full mx-auto">
+            <p class="inline-block m-1">
+                <span class="text-violet-600 text-3xl font-display font-bold">1</span>
+            </p>
+        </div>
+        <h2 class="text-center">Install the Pulumi CLI</h2>
+        <p class="text-center">
+            <span class="max-w-md inline-block">To use open source Pulumi SDK or the Pulumi Console UI you need to have the Pulumi CLI installed.</span>
```

Comment:
suggestion from Christian

```suggestion
            <span class="max-w-md inline-block">To use open-source Pulumi SDK or the Pulumi Console, you'll need to install the Pulumi CLI.</span>
```